### ISSUE TITLE
segformer: the converter shouldn't be outside the package?

### ISSUE THREAD

**Issue by johnnv1** [qubvel-org/segmentation_models.pytorch#1016]
i think this https://github.com/qubvel-org/segmentation_models.pytorch/blob/196bbe94604355b40a8982399d6f7a0809129f2f/segmentation_models_pytorch/decoders/segformer/convert_original_to_smp.py shoulnd't be outside smp package itself? in a scripts folder ou something like that (I was reviewing the need for the pillow as a dependency, and this is one of the two places it is used (the other one is in the dataset module, which only have a single sample))

**Comment by qubvel** [comment]
Yes, indeed, this one could be moved outside of the package 👍 I Would appreciate a PR in case you have the bandwidth, thank you for raising the issue
